src/main/generated/resources/data/minecolonies/crafterrecipes/sifter/diamond/dirt.json

```diff
@@ -6,12 +6,12 @@
       "item": "minecraft:dirt"
     },
     {
-      "item": "minecolonies:sifter_mesh_diamond"
+      "item": "minecolonies:sifter_mesh_diamond{Damage:0}"
     }
   ],
   "additional-output": [
     {
-      "item": "minecolonies:sifter_mesh_diamond"
+      "item": "minecolonies:sifter_mesh_diamond{Damage:0}"
```

Comment:
All of these look majorly suspicous to me.
But what the hell, i checked Mc does not handle this any better so what ever...... Mojang should have come up with a better way of doing this to be honest.

Reply:
Completely agree.